Client/Frontend/Browser/FaviconManager.swift

```diff
@@ -33,32 +33,53 @@ class FaviconManager : BrowserHelper {
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage) {
         let manager = SDWebImageManager.sharedManager()
         self.browser?.favicons.removeAll(keepCapacity: false)
-        if let url = browser?.webView!.URL?.absoluteString {
```

Comment:
Why are Favicon changes part of the spotlight work? Should this be a separate bug? Or at least a separate commit so that it clear what these changes are for?